packages/tools/viewer/src/viewer.ts

```diff
@@ -786,6 +786,7 @@ export class Viewer implements IDisposable {
             this._reflectionsIntensity = value;
 
             this._snapshotHelper.disableSnapshotRendering();
+            this._scene.environmentIntensity = this._reflectionsIntensity;

```

Comment:
You may want to use `iblIntensity` instead, `environmentIntensity` is more of a debug parameter. See #16367.

However, it seems that the environment intensity is already correctly taken into account in `_changeEnvironmentIntensity`:
```typescript
            if (this._reflectionTexture) {
                this._reflectionTexture.level = this._reflectionsIntensity;
            }
```
`this._reflectionTexture` is set as the environment texture (`this._scene.environmentTexture = this._reflectionTexture` in `_updateEnvironment`), so `this._reflectionsIntensity` should already work as expected because `this._reflectionTexture.level * scene.iblIntensity` is the final environment intensity passed to the shader.